documentation/guides/advanced-guides/multiple-browser-windows.html

```diff
@@ -0,0 +1,1221 @@
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/continuous-integration/gitlab.html"  >GitLab</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/continuous-integration/jenkins.html"  >Jenkins</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/continuous-integration/teamcity.html"  >TeamCity</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/continuous-integration/travis.html"  >Travis</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/continuous-integration/travis-and-sauce-labs.html"  >Travis + Sauce Labs</a>
+
+		
+    </li>
+
+</ul>
+		
+    </li>
+
+    <li>
+		    
+		
+
+<span>Extend TestCafe</span>
+
+		
+			<ul>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/extend-testcafe/install-plugins.html"  >Install Plugins</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/extend-testcafe/reporter-plugin.html"  >Reporter Plugin</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/extend-testcafe/browser-provider-plugin.html"  >Browser Provider Plugin</a>
+
+		
+    </li>
+
+</ul>
+		
+    </li>
+
+</ul>
+				</li>
+			</ul>
+		</div>
+	</nav>
+	
+
+	
+	
+
+	
+	
+	
+
+
+<main>
+	<article class="doc">
+		<div class="post-content">
+
+			<h1>Multiple Browser Windows</h1>
+<p>The TestCafe API includes methods that open, close, and switch between browser windows. You can test websites with pop-up windows and OAuth login forms, debug complex multi-window applications, or run multiple instances of the same web app side-by-side.</p>
+
+<p>⚠ This is a <strong>beta</strong> feature. Browser support is limited to local instances of Chrome and Firefox. You cannot capture videos or screenshots of child windows. The available functionality is subject to further revisions. Please do not use this feature in production environments.</p>
+
+<p>️🛠️ Use the <code>--disable-multiple-windows</code> CLI flag to disable support for multiple browser windows if you encounter compatibility issues with <a href="http://testcafe.devexpress.com/documentation">TestCafe v2015.1</a> tests.</p>
+<h2><a class="anchor" id="handle-client-side-window-events"></a>Handle client-side window events <a class="permalink" href="#handle-client-side-window-events">#</a></h2>
+<p>When your page launches a new window, the test automatically continues in the newly opened window. When that window is closed, the test switches back to its parent.</p>
+<div class="highlight"><pre><code class="language-js" data-lang="js"><span class="k">import</span> <span class="p">{</span> <span class="nx">Selector</span> <span class="p">}</span> <span class="k">from</span> <span class="dl">'</span><span class="s1">testcafe</span><span class="dl">'</span><span class="p">;</span>
```

Comment:
The provided example doesn't illustrate the described behavior.
Example should be like this:
```js
   test('automatically change execution context', async t => {
      await t
           .click('.google') // https://example.com
           .typeText('.google.login', '<login>')  // google authorization page
           .typeText('.google.password', '<password>') // google authorization page
           .click('submit') // google authorization page
           .expect(Selector('.logged-user').textContent).eql('user name');  // https://example.com
   });
```